docs/ga-roadmap.md

```diff
@@ -0,0 +1,114 @@
+# Collector GA Roadmap
+
+This document defines the roadmap followed by the OpenTelemetry Collector, along with tentative dates and requirements for GA (stability).
+
+In this document, the term “OpenTelemetry Collector components” refers to all the golang modules and packages that are part of the “OpenTelemetry Collector” ecosystem which include the [core](https://github.com/open-telemetry/opentelemetry-collector) and [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) components. This term does not refer to the component concept inside the “OpenTelemetry Collector”.
+
+In this document, the terms "OpenTelemetry Collector" and "Collector" both specifically refer to the OpenTelemetry Collector ecosystem’s components. These terms do not refer to the specification or the Client libraries in this document.
+
+## Current Status
+
+The OpenTelemetry Collector ecosystem right now has a lot of components that are in different stages of stability (experimental, alpha, beta, etc.). All these components have different public APIs/Interfaces (e.g. code API, configuration, etc.).
+
+A significant amount of legacy code was inherited from the Collector's ancestor [OpenCensus Service](https://github.com/census-instrumentation/opencensus-service), since then the Collector changed the internal data model and other significant changes were made.
+
+Trying to mark the entire ecosystem GA, at the same moment, will be a significant effort and will take a significant amount of time.
+
+## Proposal
+
+This document proposes a GA Roadmap based on multiple phases, where different parts of the collector will be released as stable at different moments of time.
+
+At this moment we are defining only the first two phases of the process, and the next phases will be defined at a later stage once the Collector maintainers will have better understanding of the process and implications.
+
+When other signals data models (proto definition) will be marked as stable,
+
+### Phase 1
+
+**Tentative Date:** 2021-03-31
+
+**KR:** At the end of this phase the Collector’s core API/SPI will be marked as Stable.
```

Comment:
What's KR?

Reply:
Key results

Reply:
That makes sense. Could we add an acronym legend to the doc?

Reply:
Will add a comment to this

Reply:
Done.